Library/Formula/fpc.rb

```diff
@@ -28,11 +28,18 @@ def install
 
     # Prevent non-executable audit warning
     rm_f Dir[bin/"*.rsj"]
+
+    # Generate a default fpc.cfg to facilitate search paths
+    fpcversion = `./compiler/ppc1 -iW`.chomp
```

Comment:
Will this generate the same output on every system? It won't change between Bob's Mac and Alice's Mac, if both are installed in `/usr/local`?